### PR TITLE
workflowTasks view and unit-test fix

### PR DESCRIPTION
**Pull request by srinia6**
Add workflowTasks view and fix unit-test. @RackHD/corecommitters @brianparry @keedya @dalebremner @BillyAbildgaard

### CODE REVIEW

**Comment by srinia6** [comment]
Test this please

**Comment by srinia6** [comment]
Build might have failed because workflowTask view was not present.

**Comment by srinia6** [comment]
Build Aborted,  test this please

**Comment by srinia6** [comment]
test this please

File: data/views/workflowTasks.2.0.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    <% if (hasOwnProperty("friendlyName")) { %>
+    "friendlyName": "<%=friendlyName%>",
+    <% }%>
+    <% if (hasOwnProperty("injectableName")) { %>
+    "injectableName": "<%=injectableName%>",
+    <% }%>
+    <% if (hasOwnProperty("implementsTask")) { %>
+    "implementsTask" : "<%=implementsTask%>",
+    <% }%>
```
Comment (yyscamper) [inline data/views/workflowTasks.2.0.json]: As the Task-Schema is enabled for some task definition, so there is a new key `schemaRef` need be handled here.

**Comment by yyscamper** [comment]
:+1: after adding `schemaRef`

File: data/views/workflowTasks.2.0.json (hunk 1 above)
Comment (srinia6) [inline data/views/workflowTasks.2.0.json]: @yyscamper Thank you. Will fix it.

**Comment by brianparry** [comment]
@srinia6 I'll merge after `schemaRef` is added.

**Comment by srinia6** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Comment by keedya** [comment]
test this please

**Pull request merged**